build_docker.sh

```diff
@@ -89,7 +91,9 @@ if [ -n "$STAGING_DIR" ]; then
     set -e
     GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/${REPO}/${PROJECT}.git ${STAGING_DIR}/${STAGING_CLONE_DIR}
     cd ${STAGING_DIR}/${STAGING_CLONE_DIR}
-    echo "Now in ${PWD}"
+    STAGING_ABSOLUTE_PATH=$(pwd)
+
+    echo "Now in $(pwd)"
         if [ ${PULL_REQUEST_NUMBER} ]; then
```

Comment:
the indentation here is funny although it looks like it was like that before you got here...